src/functionalTest/resources/features/F-204 - Get NoC Questions/S-204.6.td.json

```diff
@@ -28,7 +28,7 @@
   "expectedResponse": {
     "_extends_": "ACA_400_Response",
     "body": {
-      "message" : "getNoticeOfChangeQuestions.caseId: Case ID can not be empty",
+      "message" : "[[ANY_STRING_NOT_NULLABLE]]",
```

Comment:
is there a way to assert using contains (sub string) ?? 
The framework firing 3 validations using annotations.. so message order is unpredictable. 
`Eg:  "message" : "getNoticeOfChangeQuestions.caseId: Case ID can not be empty, getNoticeOfChangeQuestions.caseId: Case ID has to be 16-digits long, getNoticeOfChangeQuestions.caseId: Case ID has to be a valid 16-digit Luhn number"`

Reply:
Could this be handled in the same way as the response to `S-206.5` added in [ACA-133](https://tools.hmcts.net/jira/browse/ACA-133): where it rewrites part of the repsonse using `RestExceptionHandler.handleConstraintErrors`  (see #221) to output:

```JSON
{
    "status" : "BAD_REQUEST",
    "message" : "Case ID has to be a valid 16-digit Luhn number",
    "code" : "case-id-invalid",
    "errors" : [ "Case ID has to be a valid 16-digit Luhn number", "Case ID can not be empty", "Case ID has to be 16-digits long" ]
}
```

(i.e. you may need to strip out `getNoticeOfChangeQuestions.caseId: ` to allow the same processing to apply.)

NB: The above format includes the `code` error response value requested by EXUI under [ACA-133](https://tools.hmcts.net/jira/browse/ACA-133).

If the above works you should be able to use an `expectedResponse` matching that used in `S-205.6.td.json`.